Lab/02/Lab02_F.md

```diff
@@ -0,0 +1,170 @@
+# Soal Tutorial
+
+## Bagian A
```

Comment:
`Bagian A` vs `Part B`
Was it this way in the original .pdf file? If so, that's okay (I guess), since we try to adapt the original .pdf contents into .md as much (1:1) as possible (keeping the writer's jokes and writing style intact) 😆

Reply:
It is